bin/symfony_requirements

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env php
+<?php
+
+require_once dirname(__FILE__).'/../var/SymfonyRequirements.php';
```

Comment:
It is just used for checking if the php installation can run Symfony projects. Can be very handy when deploying it. As far as I know it is not bundled in the dependencies.